### PR TITLE
Fix style in OSDK Todo app

### PR DESCRIPTION
**Pull request by ewitkon**
1. Update style to light
3. change fake to mock
4. update language

After:
image

### CODE REVIEW

**Review by zeyadkhaled — COMMENTED** [review]
Overall I like the design changes, but few nitpicks:

1. Can we keep the removal of Tasks to be checked off instead of the trash icon. I feel this is usually how Todo apps look.
image

2. Can we try a round border for the dialogs with more padding?

image
image

**Comment by ewitkon** [comment]
> Overall I like the design changes, but few nitpicks:
> 
> 1. Can we keep the removal of Tasks to be checked off instead of the trash icon. I feel this is usually how Todo apps look.
> 
> image
> 2. Can we try a round border for the dialogs with more padding?
> 
> image

I was actually confused from the checkbox but now I do understand your point, I might ask Xinyi for her advice.

**Comment by ewitkon** [comment]
@zeyadkhaled Fixed the dialog border. 
image

Still keeping the trash button

**Comment by ewitkon** [comment]
@zeyadkhaled I reverted to checkbox with timeout

File: packages/create-app/templates/template-tutorial-todo-app/src/mocks.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ const projects: MockProject[] = [
     $apiName: "MockProject",
     $primaryKey: "1",
     id: "1",
-    name: "Fake Project",
+    name: "Mock Project",
```
Comment (zeyadkhaled) [inline packages/create-app/templates/template-tutorial-todo-app/src/mocks.ts:21]: Consistent capitalization given you changed the other project name.

```suggestion
    name: "Mock project",
```

File: examples/example-tutorial-todo-app/public/trash.svg
Comment (zeyadkhaled) [inline examples/example-tutorial-todo-app/public/trash.svg:1]: Delete this now?

File: .changeset/calm-pillows-collect.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@osdk/create-app": patch
+---
+
+Update style
```
Comment (zeyadkhaled) [inline .changeset/calm-pillows-collect.md:5]: Rename to Fix style for OSDK Todo tutorial app

File: examples/example-tutorial-todo-app/src/Home.tsx [hunk 3]
```diff
@@ -30,10 +30,10 @@ function Home() {
       <div className={css.tutorialBannerWrapper}>
         <div className={css.tutorialBanner}>
           <p className={css.tutorialBannerTitle}>
-            💡 Welcome to this tutorial!
+            💡 Welcome to To-do app tutorial!
           </p>
           <p>
-            The Todo App below has been implemented with fake in memory data.
+            The To-do App is implemented with mock in-memory data.
```
Comment (zeyadkhaled) [inline examples/example-tutorial-todo-app/src/Home.tsx:36]: Do we want to change all To-do  -> To Do to be consistent with the [AIP product](https://build.palantir.com/platform/85130d51-6134-43cf-995a-153777d83777). I forgot which one is preferred 😄 

```suggestion
            The To Do App is implemented with mock in-memory data.
```

**Review by zeyadkhaled — APPROVED** [review]

**Pull request merged**